lua/orgmode/org/mappings.lua

```diff
@@ -490,6 +490,130 @@ function OrgMappings:show_help()
   return Help.show()
 end
 
+function OrgMappings:edit_special()
+  local start_bufnr = vim.api.nvim_get_current_buf()
+  local changedtick = vim.api.nvim_buf_get_var(start_bufnr, 'changedtick')
+  local start_pos = vim.api.nvim_win_get_cursor(0)
+  local file = Files.get_current_file()
+
+  local make_temp_buf = function()
```

Comment:
Could we break this out and pass in the config? It maybe something that could be reused for [capture](https://github.com/nvim-orgmode/orgmode/pull/41)

Reply:
Not 100% sure what you mean, can you give an example?

Are you just talking about a generic way to open a buffer, that can be overridden and/or customized by the user?

Reply:
Yes, that's what was intended. Having a generic way to open a buffer could be useful elsewhere.